assets/stylesheets/kitten/components/modals/modal.scss

```diff
@@ -0,0 +1,75 @@
+/// Modal
+///
+/// @group modals
+///
+/// @example scss - usage
+///
+///   @include k-Modal((
+///     z-index: 10,
+///   ));
+///
+/// @example html
+///
+///   <div class="k-Modal k-Modal--afterOpen">
+///     <div> … </div>
+///     <div> … </div>
+///     <Button class="k-Modal__close"> … </Button>
+///   </div>
+
+@mixin k-Modal($options) {
+  $size: k-px-to-rem(16px);
```

Comment:
Est-ce qu'on peut être trouver un nom plus parlant sur ce nom de variable pour qu'elle dise quelle taille elle change ? Par exemple `$outer-gutters-size` ? Ou `$outer-margins` ? Ou `$distance-to-viewport-borders` ?